### PR TITLE
creates the new template layout for the science page

### PR DESCRIPTION
**Pull request by birdage**
@lukecampbell as discussed here is the layout im working on. i added another route for the page so we dont step on the current version.

### CODE REVIEW

**Comment by lukecampbell** [comment]
Please create a redmine issue for this under layout and navigation.

**Comment by lukecampbell** [comment]
Didn't realize this was a pull request...

File: ooiui/templates/science/index_new.html [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+    <script src="/lib/metis-menu/dist/metisMenu.js"></script>
+
+    <script src="/js/views/common/TocColView.js"></script>
+    <script src="/js/models/common/TocColModel.js"></script>
+    <script src="/js/views/common/MapView.js"></script>
+
+
+
+</head>
+
```
Comment (lukecampbell) [inline ooiui/templates/science/index_new.html]: Could you replace this with the jinja template header and footers?

Comment (birdage) [inline ooiui/templates/science/index_new.html]: yeah sure, i think i can use base as well. i just wanted to get the pull request in for the changes we discussed

**Pull request merged**